.github/workflows/test.yml

```
@@ -19,9 +19,28 @@ jobs:
 
             - name: Prepare.
               run : |
-                    pip install -r requirements.txt
-                    pip install pytest
+                  pip install -r requirements.txt
+                  pip install pytest
 
             - name: Run tests.
               run : |
-                    PYTHONPATH=$PWD pytest tests/test_topology.py
+                  cd hwloc_xml_parser
+                  PYTHONPATH=$PWD pytest tests/test_topology.py
+
+    install-as-package:
+        runs-on: [ubuntu-latest]
+        container:
+            image: python:3.9
+        steps:
+            - name: Prepare.
+              run : |
+                  apt update
+                  apt install -y hwloc
+
+            - name: Install as package.
+              run : |
+                  pip3 install git+https://github.com/uliegecsm/hwloc-xml-parser.git@make_package
```

```suggestion
                  pip3 install git+https://github.com/uliegecsm/hwloc-xml-parser.git@${{ github.sha }}
```
(to make it bullet proof)

---

OK. Thanks. Better indeed!